### ISSUE TITLE
Images from mockup

### ISSUE THREAD

**Issue by marcustisater** [postcss/postcss.org#105]
@okonet can you provide the header image used in the mockup? I would also like to know the license of it and where it's from. Thanks!

**Comment by marcustisater** [comment]
Let me rephrase that.

Can you provide all the images used for this mockup and send in an pull request for that otherwise I would just end up adding a bunch of issues for this matter.

**Comment by stephenway** [comment]
That will get exported in the design folder from the source.

**Comment by marcustisater** [comment]
That's awesome, can you make that happen @stephenway ? :wink:

**Comment by stephenway** [comment]
Sure, will come later though.

**Comment by okonet** [comment]
Here is the original image URL: http://alchemyfraternitas.ru/media/photologue/photos/gravure4.jpg

I would assume the image should be in a public domain because of its age but to make it sure I think we'll need to contact them: http://alchemyfraternitas.ru/about/

**Comment by marcustisater** [comment]
Yeah but it's best to be on the safe side - @ai can you contact them or do you have any other suggestions from #3 that we can replace?

**Comment by ai** [comment]
@okonet this imsage is 100% public domain. It is Middle Age engraving. I saw it many times in my history researches.

You can find more info by “Basilica Philosophica” keyword. It was made at 1563—1609.

**Comment by marcustisater** [comment]
Ok great, thanks! 

The images are sliced from Sketch with gulp, @stephenway is on it.

**Comment by mxstbr** [comment]
This is not yet working, so I'll get the images from the Sketch file itself.

**Comment by marcustisater** [comment]
Yup, they are not marked yet. @stephenway can you prioritize this please?

**Comment by marcustisater** [comment]
@stephenway is it possible to slice the images fromt the sketch file or is it necessary struggle to go through every  time someone would update it, otherwise I will have to close this issue.
